onnxscript/rewriter/ort_fusions/sdpa_test.py

```diff
@@ -9,6 +9,7 @@
 import unittest
 
 import numpy
+from parameterized import parameterized
```

Comment:
```suggestion
import parameterized
```
Import modules only

Reply:
Have a follow up PR to this, will fix this in that (now that this one has already been merged)